tendermint/Cargo.toml

```diff
@@ -56,3 +56,5 @@ zeroize = { version = "1.1", features = ["zeroize_derive"] }
 [dev-dependencies]
 serde_json = "1"
 tokio = "0.2"
+# Note: used in tests only:
+tendermint-lite = {path = "../tendermint-lite"}
```

Comment:
This is used in the JSON tests now. They use the mem-store it provides.

Reply:
Do they need to? Can't they just ignore the Store and operate on the returned Vec ?